src/processor.ts

```diff
@@ -70,6 +71,7 @@ const processor = new SubstrateBatchProcessor()
     .addEvent("VaultRegistry.RegisterVault", eventArgsData)
     .addEvent("VaultRegistry.IncreaseLockedCollateral", eventArgsData)
     .addEvent("VaultRegistry.DecreaseLockedCollateral", eventArgsData)
+    .addEvent("Sudo.Sudid", eventArgsData)
```

Comment:
Do we have to use an event handler for this?

Reply:
More wondering if it would be cleaner to associate with a block handler or something?

Reply:
> More wondering if it would be cleaner to associate with a block handler or something?

Are there block handlers? I wasn't able to find an example yet. If you know of an example or where in the docs I can find how to use them, please let me know.

Reply:
There is a list of handlers here: https://docs.subsquid.io/develop-a-squid/substrate-processor/data-handlers/

Reply:
I think `BlockHandler` may be what we want to use but I'm not sure exactly how it works.